content/docs/5.1/reference/faq.md

```diff
@@ -9,141 +9,143 @@ menu:
 
 ## What is TiKV?
 
-TiKV is a distributed Key-Value database that features in geo-replication, horizontal scalability, consistent distributed transactions and coprocessor support.
+TiKV is a distributed Key-value database that features in geo-replication, horizontal scalability, consistent distributed transactions, and coprocessor support.
```

Comment:
I have checked the 5.1 documents and found "key-value" is more than "Key-Value" or "Key-value". Let me revise it to "key-value" since it seems to have no difference.